source/manual/buckups.html.md

```diff
@@ -73,6 +73,16 @@ Databases are backed up to S3 using mongodump, a command-line tool that creates
 
 The environment sync replicates daily to Staging. The data is retained for 120 days.
 
+# Elasticsearch backups
+
+Elasticsearch provides the Snapshot and Restore API for creating and restoring snapshots of Elasticsearch indices. The snapshots are stored in an S3 bucket. When we create a snapshot using the API, Elasticsearch creates a copy of the index and its data and stores it in the specified repository. Elasticsearch also stores metadata about the snapshot, including the index mappings, settings, and aliases.
+
+Manual snapshots are taken overnight and stored in an S3 bucket to be used in the env sync process.
```

Comment:
Manual, but automatic, right? :D Worth linking to where the scheduled job is configured?